### ISSUE TITLE
It is empty when I run the create_data.sh to create lmdb . 

### ISSUE THREAD

**Issue by 19931991** [BVLC/caffe#6159]
error run create_data.sh , only generate the Soft connection，the size is 57 bytes , but did not genereate the actual lmdb .
this is the result of running the create_data.sh :
https://user-images.githubusercontent.com/28551961/34662842-82c71124-f48e-11e7-9482-428e10c796f0.png

and my data is :

https://user-images.githubusercontent.com/28551961/34662867-9af0cd6c-f48e-11e7-8d36-39623fcb66ec.png

the crate_data.sh is :

https://user-images.githubusercontent.com/28551961/34662884-b019dd28-f48e-11e7-94cf-2bea7713613b.png

please tell me how to solve the problem ! It troubled me so long ! Thank you very much !

**Comment by didulle** [comment]
Hi

Copy the command generated by create_data.sh and run it. You should stay in TextBoxes and than run it (./build/tools/convert_annoset --annotype............................... VOC2007_test_lmdb) in command line

**Comment by 19931991** [comment]
Thanks for your reply , I always run the create_data.sh in TextBoxes ,  but it does not work . @didulle

**Comment by 19931991** [comment]
I doubt there is a problem in the environment, but I do not know what the problem is，do you have some advances ? @didulle  Thank you very much !

**Comment by Noiredd** [comment]
Doesn't this belong to [MhLiao/TextBoxes](https://github.com/MhLiao/TextBoxes)? Caffe master does not even have a tool like `convert_annoset.cpp`.

**Comment by 19931991** [comment]
I download the code in MhLiao/TextBoxes , besides, I download the voc2007 and voc2012 , and when I run create_data.sh to create lmdb data ， it still did not succeed , the result is the same as my description - only empty soft connection. Can you tell me What environment is needed in create the lmdb ? Besides , I try to use other's lmdb (Target detection rather than text recognition) to train the network . the network runs up normally. In other words, if my data of lmdb is maked successfully , my work is done . So there is some wrong with my environment and cause this result ? Thank you very much !@Noiredd

**Comment by 19931991** [comment]
This is my problem : https://github.com/MhLiao/TextBoxes/issues/81

**Comment by tecsai** [comment]
uh, no one else encountered or resolved this issue?
Just express my issue like this.
I encountered this issue like @19931991 . I found a softlink in "example" directory under root directory whit this issue, so I delete it. Then this issue was resolved.

Update @ 2020.11.27
Once again encountered this problem, and useless with upper solution.
But I changed onelines code in db_lmdb.cpp, it's resolved.
1) mkdir(source.c_str(), 0777)
